sdk/keyvault/azure-keyvault-keys/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- If a key's ID contains a port number, this port will now be preserved in the vault URL of a
```

Comment:
Does this update description intend to be different from the rest?

Reply:
Yes; this bug was caught specifically because of how it affects `CryptographyClient`s that are created with a key ID, so I wanted to mention the bug specifically. I don't actually know of any scenario where the missing port number would affect `azure-keyvault-certificates` or `azure-keyvault-secrets`, so I just mention the change more generally while acknowledging that it was still a bug to be fixed